manifests/config.pp

```diff
@@ -234,7 +234,7 @@
     default: {}
   }
 
-  if $facts['systemd'] { # systemd fact provided by systemd module
+  if $facts['service_provider'] == 'systemd' { # systemd fact provided by systemd module
```

Comment:
This shouldn't be needed since the module depends on the systemd module. If anything, the comment should be updated.

Reply:
Ok, try to revert to facts[systemd], if don't work remove the comment.